app/assets/javascripts/components/dashboard/navigation/content_panel/signalForm.jsx

```diff
@@ -2,6 +2,8 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { reduxForm } from 'redux-form';
 import _ from 'lodash';
+import { addListenSignalData, updateListenSignalData } from '../../../../redux/modules/models/listenSignals.js';
```

Comment:
The way we've organized the components is super cumbersome to deal with. I'm going to write a PR soon that flattens everything out. Trying to reference these components from other files is never fun.